mailing_christmas/models/partner_compassion.py

```diff
@@ -29,3 +31,13 @@ class ResPartner(models.Model):
     def compute_uuid(self):
         for partner in self:
             partner.uuid = uuid.uuid4()
+
+    @api.depends('ref')
```

Comment:
Remove this decorator, only useful for stored fields to trigger the recomputation